src/main/webapp/script.js

```diff
@@ -0,0 +1,98 @@
+var zoomedIn = false;
+var aqLayerOn = true;
+var routeVisible = false;
+
+function changeBackgroundImage(imagePath) {
+    document.body.style.backgroundImage = "url('" + imagePath + "')";
+}
+
+function getImagePath() {
+    path = "";
+    if (zoomedIn) {
+        path = "images/wagga_";
+    } else {
+        path = "images/nsw_";
+    }
+
+    if (aqLayerOn) {
+        path += "gradient";
+    } else {
+        path += "plain";
+    }
+
+    if (routeVisible) {
+        path += "_route.png";
+    } else {
+        path += ".png";
+    }
+
+    return path;
+}
+function toggleAQLayer() {
```

Comment:
we gotta Keep white space consistent - 1 line between each block.

Reply:
Done